openfeature/openfeature_test.go

```diff
@@ -585,6 +585,59 @@ func TestRequirement_1_6_1(t *testing.T) {
 	}
 }
 
+// The API's `shutdown` function MUST reset all state of the API, removing all
+// hooks, event handlers, and providers.
+func TestRequirement_1_6_2(t *testing.T) {
+	defer t.Cleanup(initSingleton)
```

Comment:
Is this needed if Shutdown calls initSingleton anyways?

Reply:
It's technically not needed, but I think it makes sense to leave it in, just for consistency with the other tests.